lib/util/utils.dart

```diff
@@ -428,7 +428,7 @@ class Utils {
           fontStyle: Utils.optionalBool(style['isItalic']) == true
               ? FontStyle.italic
               : FontStyle.normal,
-          color: Utils.getColor(style['color']),
+          color: Utils.getColor(style['color']) ?? Colors.black,
```

Comment:
move hardcoded to ThemeManager and call it defaultTextColor(). This way when we implement light/dark it'll be easier to change in the theme